src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 57px;
+`;
+
+export const StyledTopBar = styled.div`
+  width: 100%;
+  height: 56px;
+  padding: 16px;
+  background-color: #2a2a2a;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+`;
+
+export const StyledHeader = styled.header`
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 19px;
+  padding-left: 53px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  color: #3c78c8;
+`;
+
+export const StyledControls = styled.div`
+  width: 115px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+export const StyledIcon = styled.img`
+  width: ${props => props.width || '24px'};
+  height: ${props => props.height || '24px'};
+`;
+
+export const StyledGradientLine = styled.div`
```

Comment:
`GradientLine` is a bit confusing. Where is it on design? :)
`BottomLine` would be a bit clearer I think... Though still sounds odd :)

Reply:
As you wish)